src/base/static/components/input-explorer/input-explorer-summary.js

```diff
@@ -94,9 +94,9 @@ const InputExplorerSummary = props => {
                 {subcategory.label + " "}
               </span>
               <span className="input-explorer-summary__summary-bar-label-percentage">
-                {isNaN(parseInt(subcategory.total / numTotal * 100))
+                {isNaN(parseInt((subcategory.total / numTotal) * 100))
                   ? 0
-                  : parseInt(subcategory.total / numTotal * 100)}%
+                  : parseInt((subcategory.total / numTotal) * 100)}%
```

Comment:
Bumping the Prettier minor version causes diffs like this one. I think this is an improvement.